packages/select/src/components/query-list/queryList.tsx

```diff
@@ -107,6 +108,13 @@ export interface IQueryListRendererProps<T> extends IProps {
      */
     handleKeyUp: React.KeyboardEventHandler<HTMLElement>;
 
+    /**
+     * Call this function to render an `item`.
+     * `QueryList` will retrieve modifiers for the item and delegate to `itemRenderer` prop for the actual rendering.
+     * The second parameter `index` is optional here; if provided, it will be passed through `itemRenderer` props.
+     */
+    renderItem: (item: T, index?: number) => JSX.Element;
```

Comment:
`itemRenderer`

Reply:
nope, this is a method given to you, not a prop you define.